index.html

```diff
@@ -12,17 +12,32 @@
         <link rel="stylesheet" href="./styles/reset.css" />
         <link rel="stylesheet" href="./styles/global.css" />
 
-        <title>Cool Web</title>
+        <title>Brens page</title>
     </head>
 
     <body>
         <header>
-            <h1>Cool Web Project</h1>
+            <h1>Brendens list of stuff</h1>
         </header>
 
-        <main></main>
+        <main>
+            <section>
+                <h1>Brenden Seifried He/Him</h1>
+                <h2>I live in <strong>Camas Washington.</strong></h2>
+                My favorite kinds of animals!
+                <ul id="animals">
+                    <li>Dog!</li>
+                    <li>Cappybara</li>
+                    <li>seals</li>
+                </ul>
+            </section>
+            <section>
+            <p>Fun Fact: I have seen many many UFO's. Starting from when I lived in Nevada to many in Washington!</p>
+            <p>I have seen lights move at high speed and change direction in the sky to full low flying craft in broad daylight.</p>
```

Comment:
Uhhmmm......this is AWESOME! I've only ever seen maybe 1-2 UFO's that I can specifically remember.